overview.md

```diff
@@ -76,7 +76,7 @@ For example, install MATLAB R2021a on a Microsoft-hosted agent, and then use the
 
 ```YAML
 pool:
-  vmImage: Ubuntu 16.04
+  vmImage: ubuntu-20.04
```

Comment:
ubuntu-latest?

Reply:
I do think it's safe/ easier to use ubuntu-latest, as it's still pointing to 20.04. I wasn't sure when azure would roll out 22.04 support since the official release was April 21. We did submit the updated install.sh script with toolbox dependencies to be deployed, which includes support for 22.04, but it's not hosted yet.

Reply:
Sounds good. Thanks for clarifying.